FreeAPS/Sources/Services/WatchManager/WatchManager.swift

```diff
@@ -19,6 +25,8 @@ final class BaseWatchManager: NSObject, WatchManager, Injectable {
     @Injected() private var tempTargetsStorage: TempTargetsStorage!
     @Injected() private var garmin: GarminManager!
 
+    @Published var date = Date()
```

Comment:
Not sure if `@Published` is required here. We could probably just inline the `Date()` where we use it.